<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PacketSenderDialog</class>
 <widget class="QDialog" name="PacketSenderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发包工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <item>
      <widget class="QPushButton" name="addProtocolButton">
       <property name="text">
        <string>添加协议</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="collapseButton">
       <property name="text">
        <string>折叠</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="expandButton">
       <property name="text">
        <string>展开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="dataPacketLayout">
       <item>
        <widget class="QLabel" name="dataPacketLabel">
         <property name="text">
          <string>数据包</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="dataPacketTreeView">
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="contentSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="detailLayout">
        <item>
         <widget class="QLabel" name="detailLabel">
          <property name="text">
           <string>详情</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="detailTreeWidget">
          <column>
           <property name="text">
            <string>属性</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>值</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Ethernet</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <item>
            <property name="text">
             <string>源MAC地址</string>
            </property>
            <property name="text">
             <string>值0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>字段1</string>
            </property>
            <property name="text">
             <string>值1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>字段2</string>
            </property>
            <property name="text">
             <string>值2</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget2">
       <layout class="QVBoxLayout" name="hexViewLayout">
        <item>
         <widget class="QLabel" name="hexViewLabel">
          <property name="text">
           <string>Hex视图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="hexViewTextEdit">
          <property name="font">
           <font>
            <family>monospace</family>
           </font>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'monospace'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;00 11 22 33 44 55 FF FF FF FF FF FF 08&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;00 28 00 01 00 00 40 11 7C CD 7F 00 00&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;00 01 04 00 04 00 00 14 00 00 00 00 00&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;00 00 00 00 00 00 00 00 00 00 00 00 00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 